### PR TITLE
Revert "Add 7zip Types"

### PR DESCRIPTION
**Pull request by EmmanuelMess**
Reverts TeamAmaze/AmazeFileManager#1170

### CODE REVIEW

**Comment by VishalNehra** [comment]
It was fully tested and was stable enough. I don't see any problem if it's been merged. You can always create an issue if you find any problems with it.

**Comment by EmmanuelMess** [comment]
Feature completeness guarantees that all features (in a release) were tested for a certain amount of time, if we keep adding features the last features added before a release won't be tested for that time, and can have bugs that weren't found but are dangerous or lie just below the surface. Also, time has to be spent in fixing a possible new set of bugs added by the PR.

**Comment by EmmanuelMess** [comment]
There is another solution, we could create a separate branch "release-3.3.0" and merge only fixes until the release. "master" branch would keep on with features and fixes merged normally.

**Comment by TranceLove** [comment]
> There is another solution, we could create a separate branch "release-3.3.0" and merge only fixes until the release. "master" branch would keep on with features and fixes merged normally.

Yes, please do.

**Comment by EmmanuelMess** [comment]
Merging to `release-3.3.0` because if this reversion is not wanted, most probably the whole branch will be deleted. `master` branch will not be affected by the reversion.

**Comment by VishalNehra** [comment]
I would like to see 7zip support on v3.3.0, since we've tested it thoroughly enough.

**Comment by EmmanuelMess** [comment]
And by that reasoning we end up with no release feature completeness and features being added to the very last day, which is what I don't want.

**Comment by TranceLove** [comment]
I'd settle focus on 3.3.0 but I'd also like to see some interim releases too (e.g. 3.3.1, 3.3.2), possibly as nightly builds including these new features... solving #1082 will help.

**Comment by VishalNehra** [comment]
I'm aware of this analogy. However we're yet to fix a few critical issues and I don't see any reason not to include a completed feature by the time we end up fixing those. It's not like I'm not working on fixing them and working on the new things.

**Comment by EmmanuelMess** [comment]
@TranceLove having releases with versions such as 3.3.x before 3.3.0 becomes confusing, nightlies should be called something else (like 3.2.1.xxxxx).

> I'm aware of this analogy. However we're yet to fix a few critical issues and I don't see any reason not to include a completed feature by the time we end up fixing those. It's not like I'm not working on fixing them and working on the new things.

I don't know the future, I don't know if everything will be fixed tomorrow, or the milestone will be removed from all issues for some reason, etc, that could bring forward the date of the release of a candidate for 3.3.0.

Do you specifically want 7zip compression to be added to 3.3.0 for a specific reason?

**Comment by VishalNehra** [comment]
7zip and marquee, both as they seem to be stable enough. It would suck to not include them as this release itself is being pushed, let alone another one which would probably take few more months. 
I'd been getting lots of messages regarding not being able to view full file names, it'd suck if I we don't include specially after taking months for the release.

**Comment by EmmanuelMess** [comment]
I would opt for patience, but I see how some people might not. If @TranceLove also thinks that it is a good idea, I won't oppose the merger of features accepted up to this time.

**Comment by TranceLove** [comment]
Not easy to answer, since it's difficult to draw a red line here and there - and I hate red lines.

Let's think in this way:
- those who can compile `master` can enjoy every new feature, and also problems the new features brought (if any). This includes we the devs, contributors and adventurers
- then the nightlies, for those who are still brave but don't want to get hands dirty. That's why #1082 is related, or if Travis allows publishing compiled apks
- finally, Google Play/F-Droid releases

If people wants cutting edge features, we at least will need to make nightlies available, and we encourage these users to give feedback.

Next, define a window for release milestones where no more new features will be accepted but only bugfixes and translation updates are accepted, for example. In fact I also didn't know when did 3.3.0 milestone was set, and when it's targeted for release...

Finally on what to include/exclude... if I need to decide what to include or not, 7zip can wait since I too think people can live without it; but marquee filenames was a long overdue problem and it affects UX by much, it would be good to include in the 3.3.0 milestone.

In short:
- define milestone and release window clearly
- only bugfixes and translation updates should be accepted for release branches after setup
- need to decide whether a feature PR should be merged to a release branch

How about that?

**Comment by EmmanuelMess** [comment]
Marquee will be merged to release-3.3.0 and master, @VishalNehra please open another PR to release-3.3.0.

**Comment by VishalNehra** [comment]
What about #1170 ? Is it going to be merged to master?

**Comment by EmmanuelMess** [comment]
It has already been merged.
